packages/flutter/lib/src/widgets/view.dart

```diff
@@ -711,6 +711,18 @@ class _MultiChildComponentElement extends Element {
     }
     return children;
   }
+
+  @override
+  void debugVisitOnstageChildren(ElementVisitor visitor, { Element? offstageAncestor }) {
+    // TODO(yiiim): The _viewElements need to be handled separately for whether they are onstage.
```

Comment:
What does this mean?

Reply:
I'm not sure yet how to handle the onstage status of other `View`s. I noticed that multi-window has not been officially released yet, this is just to avoid forgetting that elements in other `View`s need extra handling for onstage status.

Reply:
This TODO is pretty confusing then. Maybe it should be rephrased as something like "consider if _viewElements need special handling". Ideally, though, we figure out in this PR what this means for views and implement it to make sure that the solution is generic enough.